docs/workflow-executors.md

```diff
@@ -1,32 +1,29 @@
 # Workflow Executors
 
-A workflow executor is a process that conforms to a specific interface that allows Argo to perform certain actions like monitoring pod logs, collecting artifacts, managing container life-cycles, etc.
+A workflow executor runs in the pods that execute your workloads.
+It runs as both an init container and as a sidecar to the container you specify.
+It allows Argo to perform certain actions like monitoring pod logs, providing and collecting artifacts, and managing container life-cycles.
 
-## Emissary (emissary)
+Historically there were multiple available executor types, but as of 3.4 the only available executor is called `emissary`.
 
-> v3.1 and after
-
-Default in >= v3.3.
-Only option in >= v3.4.
-
-This is the most fully featured executor.
+## Emissary Executor
 
 * Reliability:
     * Works on GKE Autopilot
-    * Does not require `init` process to kill sub-processes.
-* More secure:
+    * Does not require `init` process to kill sub-processes
+* Security:
     * No `privileged` access
     * Cannot escape the privileges of the pod's service account
-    * Can [`runAsNonRoot`](workflow-pod-security-context.md).
-* Scalable:
-    * It reads and writes to and from the container's disk and typically does not use any network APIs unless resource
-    type template is used.
+    * Supports [running as non-root](workflow-pod-security-context.md)
+* Scalability:
+    * Reads and writes to and from the container's disk
+    * Typically does not use network APIs unless resource type template is used
 * Artifacts:
-    * Output artifacts can be located on the base layer (e.g. `/tmp`).
-* Configuration:
-    * `command` should be specified for containers.
+    * Output artifacts can be located on the base layer (e.g. `/tmp`)
+
+### Container Command
 
-You can determine values as follows:
+You can determine the default command for a container image using:
 
 ```bash
 docker image inspect -f '{{.Config.Entrypoint}} {{.Config.Cmd}}' argoproj/argosay:v2
```

Comment:
This image doesn't exist any more does it? I can't run the command and get a useful answer from docker anyway.